clijs/src/common/config.ts

```diff
@@ -194,6 +195,16 @@ class ConfigManager {
     (config[section] as Record<string, string>)[key] = value;
     this.saveConfig(config);
   }
+
+  private fillFallbacks() {
+    const rpc = this.getConfigValue(ConfigKeys.NilSection, ConfigKeys.RpcEndpoint);
+    if (this.getConfigValue(ConfigKeys.NilSection, ConfigKeys.CometaEndpoint) === undefined) {
+      this.updateConfig(ConfigKeys.NilSection, ConfigKeys.CometaEndpoint, rpc);
+    }
+    if (this.getConfigValue(ConfigKeys.NilSection, ConfigKeys.FaucetEndpoint) === undefined) {
+      this.updateConfig(ConfigKeys.NilSection, ConfigKeys.FaucetEndpoint, rpc);
```

Comment:
I don't think we should multiply explicit values in the config if we can do without setting them explicitly. We only add unnecessary work for users.